src/ai/backend/client/cli/params.py

```diff
@@ -55,6 +57,30 @@ def convert(self, value, param, ctx):
         return value
 
 
+class StorageProxyAddressParamCheckType(click.ParamType):
```

Comment:
How about renaming this to `CommaSeparatedKVListParamType` in accordance with `CommaSeparatedListParamType`?